test/production/edge-dynamic-code-eval/index.test.ts

```diff
@@ -0,0 +1,55 @@
+import { nextTestSetup } from 'e2e-utils'
+
+describe('Dynamic Code Evaluation (DCE)', () => {
+  const { next } = nextTestSetup({
+    skipStart: true,
```

Comment:
I think we don't need to have `skipStart` and manually build/start the app. Just need to check the `next.cliOutput` in each test

Reply:
Without `skipStart`, building and starting the prod server happens before the tests run. Our naming here is not ideal. `skipStart: false` (default) will actually run both build and run: https://github.com/vercel/next.js/blob/canary/test/lib/next-modes/next-start.ts/#L40

so `skipStart: true` will stop both of these, so that we can manually do `next.build()` in each test run.

Reply:
yeah agree, could add sth like `skipServerStart` or `buildOnly` later